lib/services/transaction-manager.js

```diff
@@ -0,0 +1,57 @@
+import Promise from 'bluebird'
+import * as constants from '../constants'
+
+export default class TransactionManager {
+  constructor (logger, client) {
+    this.client = client
+    this.logger = logger.child({ service: 'transactionManager' })
+
+    this.transactionContainer = constants.TRANSACTION_CONTAINER
```

Comment:
Why do you need to assign to field var and not using the constant directly?

Reply:
I think that the constants can be sometimes passed as object properties so they can be modified in integration tests. But not this one so I removed it dc44255